packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -118,7 +96,9 @@ const MarkerGeneHeatmap = (props) => {
           y: yOffset,
           formatter: () => {
             const shortenCellTypeLabel = numberOfRows === 1 ?
-              splitCellTypeLabel.slice(0, 1).join(`<br/>`) + `...` :
+              splitCellTypeLabel.length === 1 ?
```

Comment:
Could we refactor the 2 different method when we populate the plotLines variable, please (line 65 & 85)?
I would extract those lines to 2 well named methods.
Thanks